spec/features/signup_spec.rb

```diff
@@ -0,0 +1,12 @@
+feature "signing up" do
+  scenario "the homepage has a sign up feature" do
+    visit '/'
+    click_button 'sign up'
+    fill_in('name', with: "Coolsite")
+    fill_in('username', with: "SyedAtMakers")
+    fill_in('email', with: "help@makersacademy.com")
+    fill_in('psw', with: "getmeout")
+    click_button('Sign Up')
+    expect(page).to have_content "Welcome @username"
```

Comment:
@username doesn't get passed to a feature test so maybe use a variable or the string you passed into the username input in the test